spec/commands/exec_spec.rb

```diff
@@ -218,6 +218,18 @@
     expect(out).to include("bundler: exec needs a command to run")
   end
 
+  it "raises a helpful error when exec'ing to something outside of the bundle" do
+    install_gemfile! <<-G
+      gem "rack"
+    G
+    [true, false].each do |l|
+      bundle! "config disable_exec_load #{l}"
+      bundle "exec rake"
+      expect(err).to include "can't find executable rake for gem rake. rake is not currently included in the bundle, perhaps you meant to add it to your Gemfile?"
```

Comment:
`Couldn't find executable "rake".` should be fine unless i'm missing something? 😧